web/src/stream_topic_history.js

```diff
@@ -276,15 +276,12 @@ export function find_or_create(stream_id) {
 
 export function add_message(opts) {
     const stream_id = opts.stream_id;
-    const message_id = opts.message_id;
+    const message_id = opts.message_id ?? 0;
```

Comment:
Is undefined/0 a real possibility?

Reply:
Given the exploration I just did on your other question, I don't think so! Thanks for checking in on that.